### ISSUE TITLE
微调bge-reranker-v2-minicpm-layerwise，80G的A100卡，batchsize最大只能设置16吗

### ISSUE THREAD

**Issue by zhangying950** [FlagOpen/FlagEmbedding#1372]
数据量大概是5k组，每组里面若干个pos（1-5个）和若干个neg（2-10个）
关键参数如下：
```
num_train_epochs = 50
per_device_train_batch_size = 16
gradient_accumulation_steps = 1
train_group_size = 8
num_gpus = 2
learning_rate  = 2e-5
```

训练过程中 trainning loss 波动这么大是正常的吗？可是再往上加batch_size就oom了。
